### ISSUE TITLE
Building error on Rocky Linux 9.1: Fatal Error: build_info.inc: No such file or directory

### ISSUE THREAD

**Issue by hyperkang** [OpenRadioss/OpenRadioss#1370]
**Describe the bug**
I followed the HOWTO.md and tried to build `starter` on Rocky Linux 9.1, and got following error:

```
[ 23%] Building Fortran object CMakeFiles/starter_linux64_gf.dir/source/starter/execargcheck.F.o
/home/kang/simright-dev/OpenRadioss/starter/source/starter/execargcheck.F:1065:7:

 1065 | C-----------------------------------------------
      |       1~~~~~~~~~~~~~~~
Fatal Error: build_info.inc: No such file or directory
compilation terminated.
make[2]: *** [CMakeFiles/starter_linux64_gf.dir/build.make:24170: CMakeFiles/starter_linux64_gf.dir/source/starter/execargcheck.F.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:85: CMakeFiles/starter_linux64_gf.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

**To Reproduce**
1. Followed the HOWTO.md to set up the building environment;
2. git clone all the source files
3. cd OpenRadioss/starter
4. ./build_script.sh -arch=linux64_gf

**Expected behavior**
Got starter exec built successfully.

**Output**
Screen output as explained above

**Desktop (please complete the following information):**
 - Version [commit id: 6642450a79aecffee2e972224725d1760192080c]
- Hardware [CPU]
```
vendor_id	: GenuineIntel
cpu family	: 6
model		: 142
model name	: Intel(R) Core(TM) i7-10510U CPU @ 1.80GHz
stepping	: 12
microcode	: 0xde
cpu MHz		: 2303.998
cache size	: 8192 KB
physical id	: 0
siblings	: 1
core id		: 0
cpu cores	: 1
apicid		: 0
initial apicid	: 0
fpu		: yes
fpu_exception	: yes
cpuid level	: 22
wp		: yes
flags		: fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ss syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon nopl xtopology tsc_reliable nonstop_tsc cpuid tsc_known_freq pni pclmulqdq ssse3 fma cx16 pcid sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand hypervisor lahf_lm abm 3dnowprefetch invpcid_single ssbd ibrs ibpb stibp ibrs_enhanced fsgsbase tsc_adjust bmi1 avx2 smep bmi2 invpcid rdseed adx smap clflushopt xsaveopt xsavec xgetbv1 xsaves arat md_clear flush_l1d arch_capabilities
bugs		: spectre_v1 spectre_v2 spec_store_bypass swapgs itlb_multihit srbds mmio_stale_data retbleed eibrs_pbrsb
bogomips	: 4607.99
clflush size	: 64
cache_alignment	: 64
address sizes	: 45 bits physical, 48 bits virtual
power management:
```
- Configuration [number of OpenMP threads, and MPI tasks]
  None

**Additional context**
My Rocky Linux 9.1 is installed as virtual machine (VMWare Player 17.0), hosted on Windows 10.

**Comment by hyperkang** [comment]
Actually I tried Ubuntu 18 first, and got the same error.  Then I tried Rocky Linux 9.1.

**Comment by OlivierWienholtz** [comment]
Hi Kang !

Build system is now using Python : 
Python is versioned on some Linux distros, but the scripts can be used using Python2 or 3 as they are fairly simple.

On Ubuntu :
Pls try to install :  python-is-python3
apt-get install python-is-python3

On Rocky Linux : pls try to install : python-unversioned-command 
dnf install python-unversioned-command 

(On Rocky Linux 9.1 the package is : python-unversioned-command-3.9.14-1.el9_1.1.noarch)

**Comment by hyperkang** [comment]
> 

Hi Olivier,

I installed `python-unversioned-command`, but it doesn't help.

Below is my environment.

<img width="507" alt="image" src="https://user-images.githubusercontent.com/1835678/232443408-81021e7f-a611-4a5b-8123-2f67e5aa0252.png">

I keep getting the same error as earlier.

**Comment by OlivierWienholtz** [comment]
Hi Kang,

I found a different behave between the cmake shipped with RockyLinux 9.1 & the ones I previously tested.
I made a Fix with : 1ac71e41472cece23e43c11b25a909b22acdd1bf

Pls Retry & let me know.

**Comment by hyperkang** [comment]
> Hi Kang,
> 
> I found a different behave between the cmake shipped with RockyLinux 9.1 & the ones I previously tested. I made a Fix with : [1ac71e4](https://github.com/OpenRadioss/OpenRadioss/commit/1ac71e41472cece23e43c11b25a909b22acdd1bf)
> 
> Pls Retry & let me know.

Hi Olivier,

The fix works.  

Thank you！
